vendor/bat-native-ads/src/bat/ads/category_content_info.cc

```diff
@@ -25,6 +26,37 @@ bool CategoryContentInfo::operator!=(const CategoryContentInfo& rhs) const {
   return !(*this == rhs);
 }
 
+base::DictionaryValue CategoryContentInfo::ToValue() const {
+  base::DictionaryValue dictionary;
+
+  dictionary.SetKey("category", base::Value(category));
+  dictionary.SetKey("opt_action",
+                    base::Value(static_cast<int>(opt_action_type)));
+
+  return dictionary;
+}
+
+bool CategoryContentInfo::FromValue(const base::Value& value) {
+  const base::DictionaryValue* dictionary = nullptr;
+  if (!(&value)->GetAsDictionary(&dictionary)) {
+    return false;
+  }
+
+  const std::string* category_value = dictionary->FindStringKey("category");
+  if (category_value) {
+    category = *category_value;
+  }
+
+  const absl::optional<int> opt_action_type_optional =
+      dictionary->FindIntKey("opt_action");
+  if (opt_action_type_optional && opt_action_type_optional.has_value()) {
```

Comment:
this two conditions are checking the same:
`opt_action_type_optional ` and `opt_action_type_optional.has_value()`.  [Details here](https://source.chromium.org/chromium/chromium/src/+/main:third_party/abseil-cpp/absl/types/optional.h;l=452?q=optional.h&ss=chromium)

could we remove one of them?

Reply:
Fixed

Reply:
Fixed